lib/vsc/install/shared_setup.py

```diff
@@ -501,6 +515,30 @@ def make_release_tree(self, base_dir, files):
             self.copy_file(os.path.join(REPO_BASE_DIR, fn), os.path.join(base_dir, fn))
 
 
+class vsc_sdist_rpm(vsc_sdist):
+    """Manipulate the shebang in all scripts"""
+
+    def make_release_tree(self, base_dir, files):
+        vsc_sdist.make_release_tree(self, base_dir, files)
+
+        if self.distribution.has_scripts():
+            # code based on sdist add_defaults
+            build_scripts = self.get_finalized_command('build_scripts')
+            scripts = build_scripts.get_source_files()
+
+            log.info("scripts to check for shebang %s" % (scripts))
+            pyshebang_reg = re.compile(r'\A'+SHEBANG_ENV_PYTHON)
+            for fn in scripts:
+                first_line = open(os.path.join(base_dir, fn)).readline()
+                if pyshebang_reg.search(first_line):
+                    log.info("going to adapt shebang for script %s" % fn)
+                    dest, code = self._recopy(base_dir, fn)
+                    code = pyshebang_reg.sub(SHEBANG_PYTHON_E, code)
```

Comment:
true, it would just prepend. it was assuming the unittest ran before, which i shouldn't do :smile: